perf/benchmark/linkerd/fortio.yaml

```diff
@@ -0,0 +1,200 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
```

Comment:
This file seems identical to the one in benchmarks.
Can we reuse it? If it is different I would like to know how?

Reply:
The deployment has linkerd injection enabled. https://github.com/askmeegs/tools/blob/linkerd-install/perf/benchmark/linkerd/fortio.yaml#L15

Reply:
https://github.com/istio/tools/blob/master/perf/benchmark/templates/fortio.yaml#L87   It is already there as an option in the helm template.